miner/scroll_worker.go

```diff
@@ -690,11 +690,19 @@ func (w *worker) processTxn(tx *types.Transaction) (bool, error) {
 	w.current.txs = append(w.current.txs, tx)
 	w.current.receipts = append(w.current.receipts, receipt)
 
+	txRlp, _ := tx.MarshalBinary()
```

Comment:
what about handling errors, all it doesn't need here?

Reply:
When we goes here, the tx should be valid and no error should exists (unless oom). If we need to handle this error, the best should just panic the node.

Reply:
got it.